addons/point_of_sale/static/src/app/screens/receipt_screen/receipt_screen.js

```diff
@@ -83,6 +84,18 @@ export class ReceiptScreen extends Component {
         const { name, props } = this.nextScreen;
         this.pos.showScreen(name, props);
     }
+
+    async generateTicketImage(isBasicReceipt = false) {
+        return await this.renderer.toJpeg(
+            OrderReceipt,
+            {
+                data: this.pos.orderExportForPrinting(this.pos.get_order()),
+                formatCurrency: this.env.utils.formatCurrency,
+                ...(isBasicReceipt && { basic_receipt: true }),
```

Comment:
```suggestion
                 basic_receipt: isBasicReceipt,
```
Why wouldn't this work?